docs/exec.md

```diff
@@ -13,14 +13,14 @@ nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-001
 nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-002 --tma
 ```
 
-Additional flags can be used to control inclusion and exclusion of individual modules in the pipeline.
+By default, the pipeline starts from the registration step. Use `--start-at` and `--stop-at` flags to execute any contiguous section of the pipeline instead. (Note that starting at any step beyond registration requires pre-computed output of the previous steps placed at the correct location in the project directory.)
```

Comment:
Be more explicit about the vocabulary of values allowed for `--start-at` and `--stop-at`, i.e., the directory names described in the Directory Structure section (dir.md).